patch/kconfig-inclusions

```diff
@@ -107,6 +115,12 @@ CONFIG_SENSORS_LM25066=m
 CONFIG_SENSORS_UCD9000=m
 CONFIG_SENSORS_UCD9200=m
 CONFIG_THERMAL_OF=y
+CONFIG_SPI_AMD=m
+CONFIG_PINCTRL_AMD=y
+CONFIG_EDAC_AMD64=m
+CONFIG_AMD_XGBE_DCB=y
+CONFIG_AMD_XGBE_HAVE_ECC=y
+CONFIG_USB_NET_CDCETHER=m
```

Comment:
Are `CONFIG_USB_NET_DRIVERS`, `CONFIG_USB_USBNET`, and `CONFIG_USB_NET_CDCETHER` actually needed? The last two appear to be for older or specialized NICs.